services/gateway/apis/http-version2/methods/accounts.js

```diff
@@ -40,6 +41,27 @@ module.exports = {
 			enum: ['balance:asc', 'balance:desc', 'rank:asc', 'rank:desc'],
 		},
 	},
+	get schema() {
+		const accountSchema = {};
+		accountSchema[this.swaggerApiPath] = { get: {} };
+		accountSchema[this.swaggerApiPath].get.tags = this.tags;
+		accountSchema[this.swaggerApiPath].get.parameters = transformParams(
+			'accounts',
+			this.params,
+		);
+		accountSchema[this.swaggerApiPath].get.summary = 'Requests account data';
+		accountSchema[this.swaggerApiPath].get.description = `Returns account data\n RPC=> ${this.rpcMethod}`;
```

Comment:
Can you make the description based on a shared function that operates as a template?
It would accept an object with two properties, rpc method and description and allow to customise it in one place.

Reply:
Yes, I will do it right away.

Reply:
Done!